app/mailers/admin_mailer.rb

```diff
@@ -47,7 +47,17 @@ def edited_comment_notification(comment_id)
 
   # Sends a spam report
   def send_spam_alert(spam)
-    @spam = spam
+    @users = User.where(id: spam.keys).to_a
+    abort_delivery if @users.empty?
```

Comment:
In Rails 5, can we call `mail` conditionally instead of using `BulletproofMailer`? https://stackoverflow.com/a/13276740

Reply:
Ooh, good point! Looks like the `BulletproofMailer` workaround is no longer necessary.

Reply:
Can you add a TODO remove at the top of `BulletproofMailer`?

Reply:
It's still used for `UserMailer#batch_subscription_notification`, so it can't be removed immediately. Can I maybe add a bug report for removing it and updating the subscription mailer, instead?

Reply:
Please and thank you, that's better than a TODO.

Reply:
I added the issue: https://otwarchive.atlassian.net/browse/AO3-5708

Reply:
Thanks!